src/framework/audio/internal/audiothread.h

```diff
@@ -45,9 +44,13 @@ class AudioThread
     //! use if you don't want to use internal thread
     void loopBody();
 
+    std::shared_ptr<rpc::QueuedRpcChannel> channel() const;
```

Comment:
Better to add an alias to this:

using QueuedRpcChannelPtr = std::shared_ptr<rpc::QueuedRpcChannel>

Reply:
added